File: c/2-control structures/workout/csw-2.c
#include <stdio.h>
int main()
{
    int a;
    scanf("%d\n",&a);
    if(a>=18)
    {
        printf("Eligible");
        
    }
    else if(a<0)
        printf("Invalid input");

    else
        printf("Not eligible");
        return 0;
}

File: java/3-control-statements/workout/csw-2.java
import java.util.Scanner;
public class A{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        String binary = sc.nextLine();
        int decimal = Integer.parseInt(binary,2);
        if(decimal>0){
            System.out.println(decimal);
        }
        else{
            System.out.println("Invalid input");
        }
        
    }
}